src/pages/nieuwePost/NieuwePost.jsx

```diff
@@ -0,0 +1,136 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+function NieuwePost() {
+    const [formData, setFormData] = useState({
+        title: '',
+        subtitle: '',
+        content: '',
+        author: ''
+    });
+    const [errors, setErrors] = useState({});
+
+    const navigate = useNavigate();
+
+    const validate = () => {
```

Comment:
in de uitwerkingen hebben ze de foutmeldingen gedaan door "required" toe te voegen bij de input. is een stuk korter, maar hoe jij het doet heb je wel controle over het bericht.